docs/tutorials/unsupervised_learning/gan.md

```diff
@@ -1,43 +1,43 @@
-# Generative Adversarial Networks
+# Generative Adversarial Networks (GAN)
 
 GANs are an application of unsupervised learning - you don't need labels for your dataset in order to train a GAN.
```

Comment:
This sentence seems slightly off.
Wikipedia's version (modified) sounds better I think:
"Generative adversarial networks (GANs) are a class of algorithms used in unsupervised learning..".